pkg/apis/networking/validation/validation.go

```diff
@@ -214,10 +216,8 @@ func ValidateIngressCreate(ingress *networking.Ingress, requestGV schema.GroupVe
 	allErrs := field.ErrorList{}
 	var opts IngressValidationOptions
 	opts = IngressValidationOptions{
-		// TODO(robscott): Remove regex validation for 1.19.
-		requireRegexPath: true,
-		// TODO(cmluciano): Allow resource backend for 1.19.
-		allowResourceBackend: false,
+		requireRegexPath:     false,
+		allowResourceBackend: true,
```

Comment:
Instead of keeping these options can we just remove them entirely?

Reply:
yes, opened https://github.com/kubernetes/kubernetes/pull/91034 to do that independently